### PR TITLE
Changing use of block.number to block.timestamp

### PR DESCRIPTION
**Pull request by fleupold**
As title. Using 3 minutes now instead of 20 blocks.

### CODE REVIEW

**Comment by fleupold** [comment]
Closes #59

**Review by josojo — APPROVED** [review]
good work

File: test/snapp_auction.js [hunk 1]
```diff
@@ -89,13 +90,13 @@ contract("SnappAuction", async (accounts) => {
       await setupEnvironment(MintableERC20, instance, token_owner, [user_1], 2)
       await instance.placeSellOrder(1, 2, 1, 1, { from: user_1 })
 
-      await waitForNBlocks(21, owner)
+      await waitForNSeconds(601, owner)
```
Comment (bh2smith) [inline test/snapp_auction.js]: Is there any reason you didn't use 60*3 + 1  for 3 minutes and 1 second?

File: test/utilities.js [hunk 2]
```diff
@@ -100,11 +100,24 @@ const setupEnvironment = async function(token_artifact, contract, token_owner, a
...
+    })
+  })
+}
+
 // Wait for n blocks to pass
-const waitForNBlocks = async function(numBlocks, authority, web3Provider=web3) {
-  for (let i = 0; i < numBlocks; i++) {
-    await web3Provider.eth.sendTransaction({from: authority, to: authority, value: 1})
-  }
+const waitForNSeconds = async function(seconds, authority, web3Provider=web3) {
```
Comment (bh2smith) [inline test/utilities.js]: Sad too see wait for N blocks being entirely removed. I guess it really is no longer needed here.

File: scripts/wait_seconds.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const { waitForNSeconds } = require("../test/utilities.js")
```
Comment (bh2smith) [inline scripts/wait_seconds.js:1]: We will need to remember to change the `mine_blocks` usages in dex-services to `wait_seconds`

**Review by bh2smith — APPROVED** [review]
Wow, that was no nearly as bad as I thought it might be! No more mining 🗳, its about ⌚️ . Maybe Double check your usage of 601 seconds in the unit test (since `60 * 3 + 1 = 181`)

File: test/snapp_auction.js (hunk 1 above)
Comment (fleupold) [reply test/snapp_auction.js]: Great catch! Who could have known that 3 minutes is not 600 seconds.

🎣

**Pull request merged**